openstack/aad/v1/rules/requests.go

```diff
@@ -0,0 +1,100 @@
+package rules
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+// BatchCreateOpts is the structure required by the BatchCreate method to batch create at least one forward rule.
+type BatchCreateOpts struct {
+	// Forwarding rules created in batches.
+	Rules []RuleOpts `json:"rules,omitempty"`
+}
+
+// RuleOpts is the object that represents the forwarding rule configuration structure.
+type RuleOpts struct {
+	// The forward protocol.
+	ForwardProtocol string `json:"forward_protocol" required:"true"`
+	// The forward port.
+	// The valid value is range from 1 to 65535.
+	ForwardPort int `json:"forward_port" required:"true"`
+	// The source port.
+	// The valid value is range from 1 to 65535.
+	SourcePort int `json:"source_port" required:"true"`
+	// The source IP list, separate the IPs with commas.
+	SourceIp string `json:"source_ip" required:"true"`
+}
+
+var requestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
+
+// BatchCreate is a method to to batch create at least one forward rule using given parameters.
+func BatchCreate(c *golangsdk.ServiceClient, InstnaceId, ip string, opts BatchCreateOpts) (*BatchResp, error) {
```

Comment:
s/InstnaceId/**i**nstnaceId/g
the var should in first-lowercase format

Reply:
done.